slack-connector/internal/registration/payloadbuilder.go

```diff
@@ -0,0 +1,74 @@
+package registration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/kyma-incubator/hack-showcase/slack-connector/internal/apperrors"
+)
+
+const (
+	specificationURL          = "https://raw.githubusercontent.com/kyma-incubator/hack-showcase/slack-connector-boilerplate/slack-connector/internal/registration/configs/slackopenapi.json" // to edit after merge
```

Comment:
Consider moving those URLs to config / params / env

Reply:
But we will leave it as it is. This url will stay constant throughout lifetime of app.

Reply:
```suggestion
	specificationURL          = "https://raw.githubusercontent.com/kyma-incubator/hack-showcase/master/slack-connector/internal/registration/configs/slackopenapi.json"
```